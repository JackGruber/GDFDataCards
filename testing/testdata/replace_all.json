{
    "id": "ocjUa4aH72ba",
    "gameSystem": "gf",
    "armyId": "4k5amkxoybdiqotm",
    "armyIds": [
        "4k5amkxoybdiqotm"
    ],
    "armyFaction": null,
    "armyName": "Robot Legions",
    "modified": "2024-04-01T19:58:13.975Z",
    "saveVersion": 3,
    "favourite": false,
    "armyVersions": [
        {
            "version": "3.2.0",
            "armyId": "4k5amkxoybdiqotm"
        }
    ],
    "listPoints": 110,
    "list": {
        "id": "ocjUa4aH72ba",
        "name": "Necrons Combat Patrol 1.2k",
        "points": 1200,
        "isCloud": true,
        "modified": "2024-04-01T19:58:13.975Z",
        "gameSystem": "gf",
        "competitive": true,
        "pointsLimit": 1200,
        "unitPreview": null,
        "campaignMode": false,
        "selectedUnitId": "FnPS0",
        "undoUnitRemove": [
            {
                "id": "wvgi7_R",
                "xp": 0,
                "cost": 175,
                "name": "Eternals",
                "size": 5,
                "notes": null,
                "valid": true,
                "armyId": "4k5amkxoybdiqotm",
                "sortId": 6,
                "traits": [],
                "defense": 2,
                "loadout": [
                    {
                        "id": "_WpwGRUk",
                        "name": "CCW",
                        "type": "ArmyBookWeapon",
                        "count": 5,
                        "label": "CCW (A1)",
                        "range": 0,
                        "attacks": 1,
                        "weaponId": "4o5CBBZC",
                        "specialRules": [],
                        "originalCount": 5
                    },
                    {
                        "id": "dT086NsK",
                        "name": "Flux Carbine",
                        "type": "ArmyBookWeapon",
                        "count": 5,
                        "label": "Flux Carbine (18\", A2, Flux)",
                        "range": 18,
                        "attacks": 2,
                        "weaponId": "dT086NsK",
                        "specialRules": [
                            {
                                "key": "flux",
                                "name": "Flux",
                                "modify": false,
                                "rating": ""
                            }
                        ],
                        "originalCount": 5
                    }
                ],
                "quality": 3,
                "combined": false,
                "upgrades": [
                    "F1"
                ],
                "equipment": [
                    {
                        "id": "_WpwGRUk",
                        "name": "CCW",
                        "type": "ArmyBookWeapon",
                        "count": 5,
                        "label": "CCW (A1)",
                        "range": 0,
                        "attacks": 1,
                        "weaponId": "4o5CBBZC",
                        "specialRules": [],
                        "originalCount": 5
                    },
                    {
                        "id": "dT086NsK",
                        "name": "Flux Carbine",
                        "type": "ArmyBookWeapon",
                        "count": 5,
                        "label": "Flux Carbine (18\", A2, Flux)",
                        "range": 18,
                        "attacks": 2,
                        "weaponId": "dT086NsK",
                        "specialRules": [
                            {
                                "key": "flux",
                                "name": "Flux",
                                "modify": false,
                                "rating": ""
                            }
                        ],
                        "originalCount": 5
                    }
                ],
                "joinToUnit": null,
                "selectionId": "_xivy",
                "specialRules": [
                    {
                        "key": "regeneration",
                        "name": "Regeneration",
                        "rating": ""
                    },
                    {
                        "key": "robot",
                        "name": "Robot",
                        "rating": ""
                    },
                    {
                        "key": "slow",
                        "name": "Slow",
                        "rating": ""
                    }
                ],
                "disabledSections": [],
                "selectedUpgrades": [],
                "disabledUpgradeSections": []
            }
        ],
        "activationCount": 1,
        "modelCount": 5,
        "units": [
            {
                "id": "njiIkJi",
                "armyId": "4k5amkxoybdiqotm",
                "customName": "Necronkrieger",
                "selectionId": "Q8tjV",
                "selectedUpgrades": [
                    {
                        "instanceId": "FszmpH6cX",
                        "upgradeId": "sPruWe5",
                        "optionId": "r_yHp"
                    }
                ],
                "combined": false,
                "joinToUnit": null,
                "xp": 0,
                "traits": [],
                "notes": null
            }
        ]
    }
}