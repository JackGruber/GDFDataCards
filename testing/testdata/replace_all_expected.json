{
    "armyId": "4k5amkxoybdiqotm",
    "gameSystem": "gf",
    "gameSystemId": 2,
    "armyName": "Robot Legions",
    "listPoints": 110,
    "listName": "Necrons Combat Patrol 1.2k",
    "units": [
        {
            "type": "Warriors",
            "name": "Necronkrieger",
            "armyId": "4k5amkxoybdiqotm",
            "id": "njiIkJi",
            "cost": 115,
            "defense": 4,
            "quality": 4,
            "upgrades": [
                "E1"
            ],
            "size": 5,
            "notes": null,
            "weapons": [
                {
                    "attacks": 1,
                    "count": 5,
                    "name": "CCW",
                    "specialRules": []
                },
                {
                    "attacks": 1,
                    "count": 5,
                    "name": "Reaper Rifle",
                    "range": 18,
                    "specialRules": [],
                    "ap": "2"
                }
            ],
            "specialRules": [
                {
                    "id": 582,
                    "key": "robot",
                    "name": "Robot",
                    "label": "Robot"
                },
                {
                    "id": 964,
                    "key": "self-repair",
                    "name": "Self-Repair",
                    "label": "Self-Repair"
                },
                {
                    "key": "slow",
                    "name": "Slow",
                    "rating": "",
                    "label": "Slow"
                }
            ],
            "upgradeCost": [
                -5
            ]
        }
    ]
}